src/soilId/soilIdSlice.ts

```diff
@@ -17,6 +17,7 @@
 
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { setUsers } from 'terraso-client-shared/account/accountSlice';
+import { SoilIdFailureReason } from 'terraso-client-shared/graphqlSchema/graphql';
```

Comment:
Issue: Wait ok where is this? I don't see it defined anywhere 😅

Reply:
i guess technically it's defined:
-  here in the backend PR: https://github.com/techmatters/terraso-backend/pull/1345/files#diff-e5ed61d163d5ffb8bb0409bf1896c0e6fffcca7aac0d4be14fe931369381cca9R105-R111
- which autogenerates these in the `.graphql` schema: https://github.com/techmatters/terraso-backend/pull/1345/files#diff-e820e1a84e67dbabf615830da710c0566c9095335eb5bb2de5292f5d6d6fc868R233-R240
- which is included in client-shared via npm dependency
- and then gets turned into a typescript to by `client-shared`'s codegen

Reply:
Ah gotcha, thanks